File: app/src/main/java/com/neome/api/home/base/dto/DtoMessageReplyPayloadVideo.java
// neome.ai API - do not change
//

package com.neome.api.home.base.dto;

import com.neome.api.meta.base.Types.MediaIdImage;
import com.neome.api.meta.base.Types.MediaIdVideo;

@SuppressWarnings("unused")
public class DtoMessageReplyPayloadVideo extends DtoMessageReplyPayload
{
  public long durationMs;

  public MediaIdImage mediaId;

  public MediaIdImage mediaIdBlurImage;

  public MediaIdVideo mediaIdVideo;

  public String primaryColor;
}

File: app/src/main/java/com/neome/api/home/base/dto/DtoMessageReplyPayloadVideo.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.base.dto

import com.neome.api.home.base.dto.DtoMessageReplyPayload
import com.neome.api.meta.base.Types.MediaIdImage
import com.neome.api.meta.base.Types.MediaIdVideo

interface DtoMessageReplyPayloadVideo : DtoMessageReplyPayload
{
  val durationMs: Long?
  val mediaId: MediaIdImage
  val mediaIdBlurImage: MediaIdImage
  val mediaIdVideo: MediaIdVideo
  val primaryColor: String
}